src/main/java/wedlog/address/logic/Messages.java

```diff
@@ -112,7 +113,19 @@ public DisplayBuilder addOptional(String fieldName, Optional<? extends Object> f
             return this;
         }
 
+        private DisplayBuilder addDietaryRequirements(Set<DietaryRequirement> dietaryRequirements) {
```

Comment:
Could you leave a short comment describing what this method does? Same for `addTags`.